src/errors/runtime/templates.js

```diff
@@ -7,101 +7,100 @@ import { RUNTIME_ERRORS } from '../types';
-    [RUNTIME_ERRORS.clientFunctionCodeIsNotAFunction]:                   '{#instantiationCallsiteName} code is expected to be specified as a function, but {type} was passed.',
-    [RUNTIME_ERRORS.selectorInitializedWithWrongType]:                   '{#instantiationCallsiteName} is expected to be initialized with a function, CSS selector string, another Selector, node snapshot or a Promise returned by a Selector, but {type} was passed.',
+    [RUNTIME_ERRORS.cannotSetConcurrencyWithCDPPort]:                    'The concurrency factor can not include the CDP port.',
+    [RUNTIME_ERRORS.portsOptionRequiresTwoNumbers]:                      'The "--ports" argument accepts two values at a time.',
+    [RUNTIME_ERRORS.portIsNotFree]:                                      'Port {portNum} is occupied by another process.',
+    [RUNTIME_ERRORS.invalidHostname]:                                    'Cannot resolve hostname "{hostname}".',
+    [RUNTIME_ERRORS.cannotFindSpecifiedTestSource]:                      'Cannot find a test file at "{path}".',
+    [RUNTIME_ERRORS.clientFunctionCodeIsNotAFunction]:                   'Cannot execute a clientFunction becuase {#instantiationCallsiteName} is {type}, and not a function.',
+    [RUNTIME_ERRORS.selectorInitializedWithWrongType]:                   'Cannot initialize a Selector because {#instantiationCallsiteName} is {type}, and not one of the following: a CSS selector string, a Selector object, a node snapshot, a function, or a Promise returned by a Selector.',
     [RUNTIME_ERRORS.clientFunctionCannotResolveTestRun]:                 "{#instantiationCallsiteName} cannot implicitly resolve the test run in context of which it should be executed. If you need to call {#instantiationCallsiteName} from the Node.js API callback, pass the test controller manually via {#instantiationCallsiteName}'s `.with({ boundTestRun: t })` method first. Note that you cannot execute {#instantiationCallsiteName} outside the test code.",
     [RUNTIME_ERRORS.regeneratorInClientFunctionCode]:                    `{#instantiationCallsiteName} code, arguments or dependencies cannot contain generators or "async/await" syntax (use Promises instead).`,
-    [RUNTIME_ERRORS.invalidClientFunctionTestRunBinding]:                'The "boundTestRun" option value is expected to be a test controller.',
-    [RUNTIME_ERRORS.invalidValueType]:                                   '{smthg} is expected to be a {type}, but it was {actual}.',
-    [RUNTIME_ERRORS.unsupportedUrlProtocol]:                             'The specified "{url}" test page URL uses an unsupported {protocol}:// protocol. Only relative URLs or absolute URLs with http://, https:// and file:// protocols are supported.',
+    [RUNTIME_ERRORS.invalidClientFunctionTestRunBinding]:                'Cannot resolve the "boundTestRun" option becuase its value is not a test controller.',
```

Comment:
```suggestion
    [RUNTIME_ERRORS.invalidClientFunctionTestRunBinding]:                'Cannot resolve the "boundTestRun" option because its value is not a test controller.',
```